components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -50,8 +54,13 @@ class RewardsServiceObserver : public base::CheckedObserver {
                              brave_rewards::ContentSiteList) {};
   virtual void OnPublisherBanner(brave_rewards::RewardsService* rewards_service,
                                  const brave_rewards::PublisherBanner banner) {};
+  virtual void OnGetPublisherActivityFromUrl(
```

Comment:
we specifically cannot do this because it exposes ledger::* types. Only ContentSiteList/ContentSite should ever be exposed in the public rewards_service API

Reply:
OK I see. I think this was originally implemented in the private observer, but @SergeyZhukovsky asked to have it moved to the public observer for use with Android (I can't recall exactly why).